doc/gov/chapters/chapter05.md

```diff
@@ -33,8 +33,19 @@ Being on the voting representative list **doesn't** mean the right to vote. The
 
 <sup>2)</sup>The criteria will be checked and the voting representative list will be updated prior any voting event when the voting process kicks in.
 
-<sup>3)</sup> Individuals who don't present an organisation will not have voting right.
+<sup>3)</sup> Individuals who don't represent an organisation will not have voting right.
 
 <a name="5.3"></a>
 ## 5.3 Voting Process
-
+ 
+ In Scenarios where Consensus cannot be reached, a voting event will be triggered by Technical/Governance Steering Leads (depending on if the event corresponds to a technical or governance matter). 
```

Comment:
... depending on whether the event ...

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/897/commits/b8c146de14773c51b36a811176daaabddfeb47f7